src/plugins/dashboard/public/dashboard_app/top_nav/share/show_share_modal.tsx

```diff
@@ -169,7 +169,9 @@ export function ShowShareModal({
     unsavedStateForLocator = {
       query: unsavedDashboardState.query,
       filters: unsavedDashboardState.filters,
-      controlGroupInput: unsavedDashboardState.controlGroupInput as SerializableControlGroupInput,
+      controlGroupState: panelModifications?.[
+        PANELS_CONTROL_GROUP_KEY
+      ] as DashboardLocatorParams['controlGroupState'],
```

Comment:
This is tricky, because the runtime state has changed shape. The runtime state no longer has explicitInput. Devon has stated that its ok to break URLs in these cases.

Reply:
Hmm... Yeah, that's fair. If we are okay with old links being broken, I suppose that is fine :) What do you think about getting ahead of this and creating a known issue? I don't think it's large enough to be considered a breaking change, but I can definitely see "Controls are broken!" SDHs coming from this change 🙈

Reply:
chatted offline. thanks for opening the known issue